src/main/scala/com/github/danielwegener/intellij/cucumber/scala/CucumberScalaExtension.scala

```diff
@@ -52,8 +53,16 @@ class CucumberScalaExtension extends AbstractCucumberExtension {
 
   @NotNull override def getGlues(@NotNull file: GherkinFile, jGluesFromOtherFiles: JavaSet[String]): JavaCollection[String] = {
     // never called? wtf
-    LOG.debug("GET GLUES CALLED")
-    Collections.emptyList()
+    import scala.collection.JavaConverters._
```

Comment:
Could you elaborate on und which circumstances this method it is called?

Reply:
it's called when we right click on the feature and select run.
The glue package this returns is in the run configuration generated

Reply:
From searching forums it might just be called in the community edition. That is what we are using.